tests/integration/targets/vyos_l3_interfaces/tests/cli/parsed.yaml

```diff
@@ -0,0 +1,32 @@
+---
+- debug:
+    msg: START vyos_l3_interfaces parsed integration tests on connection={{ ansible_connection
+      }}
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate.yaml
+
+- block:
+
+    - name: Parse the running config
+      register: l3_interfaces_facts
+      vyos.vyos.vyos_facts:
+        gather_subset:
+          - default
+        gather_network_resources:
+          - l3_interfaces
+
+    - name: Provide the running configuration for parsing (config to be parsed)
+      register: result
+      vyos.vyos.vyos_l3_interfaces:
+        running_config: "{{ lookup('file', '_parsed_config.cfg') }}"
+        state: parsed
+
+    - name: Assert that correct parsing done
+      assert:
+        that: "{{ ansible_facts['network_resources']['l3_interfaces'] | symmetric_difference(result['parsed'])\
```

Comment:
Both RHS and LHS are generated by the same piece of code and if we pass the same running-config to both, it will always result in True, regardless of whether the list of dicts generated are correct. 

This should be hardcoded or at least use a previously hardcoded value.